taqueria-plugin-ligo/common.ts

```diff
@@ -14,7 +15,7 @@ export type IntersectionOpts = CompileOpts & TestOpts;
 type UnionOpts = CompileOpts | TestOpts;
 
 // Points to the latest stable version. Needs to update this as part of our release process
-export const LIGO_DOCKER_IMAGE = 'ligolang/ligo:0.51.0';
+export const LIGO_DOCKER_IMAGE = getDockerImage('ligolang/ligo:0.51.0', 'TAQ_LIGO_IMAGE');
```

Comment:
Doesn't sound like good practice to have a function return a value that may be different with each invocation and assign it to a global constant.

Reply:
I see this as effectively the same as a constant value - it just can be temporary overwritten for a particular invocation. The value doesn't change over the lifetime of the execution, so I thought this was fine. Did you have something else in mind? I'm open to other approaches.

Reply:
I think it'd be better to use `getDockerImage()` when we are using the image. And then the function would depend on LIGO_DOCKER_IMAGE as a parameter.

e.g. `getDockerImage(LIGO_DOCKER_IMAGE, 'TAQ_LIGO_IMAGE')`

Reply:
OK, I'll adjust. Thanks!